community-roles.md

```diff
@@ -96,7 +96,7 @@ This team drives the direction, values and governance of the overall project.
 It is important to us that its members come from a diverse background of companies and organizations.
 Ensuring that oversight of the project is not controlled by one company or organization.
 
-**Defined by:** entry in [flux2 MAINTAINERS file](https://github.com/fluxcd/flux2/blob/main/MAINTAINERS), and in the `@fluxcd/flux2-maintainers` GitHub team.
+**Defined by:** entry in [flux2 MAINTAINERS file](https://github.com/fluxcd/flux2/blob/main/MAINTAINERS), and in the `@fluxcd/core-maintainers` GitHub team.
```

Comment:
I would like to see this file moved to a neutral place.

Reply:
`https://github.com/fluxcd/community/blob/main/CORE-MAINTAINERS` maybe?

Reply:
And remove `https://github.com/fluxcd/flux2/blob/main/MAINTAINERS)` then?

Reply:
We can't move it, each repo must contain a MAINTAINERS file, but we can link from flux2 MAINTAINERS file to community CORE-MAINTAINERS one.

Reply:
Ok.

Reply:
:+1: